charmcraft.yaml

```diff
@@ -33,13 +33,9 @@ parts:
 
 subordinate: true
 
-peers:
-  storage-peers:
-    interface: storage_peers
-
 requires:
-  fs-share:
-    interface: fs_share
+  filesystem:
+    interface: filesystem_info
```

Comment:
`filesystem` seems good for the relation name, but I'm still unsure about the name for the interface. `filesystem_info` makes sense to me, but I'm open to other proposals.

Reply:
I like the name `filesystem_info`! I think it pretty accurately captures that the integration is sharing info about the filesystem.